application/container-builder/git/clone.go

```diff
@@ -3,9 +3,10 @@ package git
 import "github.com/go-git/go-git/v5"
 
 // Eventually clone should support cloning a specific branch
-func Clone(repoName string) (*git.Repository, error) {
-	return git.PlainClone(repoName, false, &git.CloneOptions{
-		URL:               repoName,
+func Clone(repo, path string) (*git.Repository, error) {
```

Comment:
think we can skip validation for now as git.PlainClone would error out if it's unable to pull the public repo